community/gsoc_responsibilities.md

```diff
@@ -2,7 +2,9 @@ Responsibilities of coala’s GSoC Participants
 =============================================
 
 This Document hold the responsibilities we expect coala’s GSoC participants to
-fulfil.
+fulfil.  
```

Comment:
you are adding trailing whitespace to this line.
Please dont add trailing whitespace to any line.